apps/ima/index.js

```diff
@@ -31,11 +31,145 @@ module.exports = {
     },
     '/who-are-you': {
       behaviours: SaveFormSession,
+      forks:[
+        {
+          target:'/in-the-uk',
+          condition:{
+            field:'who-are-you',
+            value: 'person-named'
+          }
+        },
+        {
+          target:'/legal-representative-details',
+          condition:{
+            field:'who-are-you',
+            value: 'has-legal-representative'
+          }
+        },
+        {
+          target:'/someone-else',
+          condition:{
+            field:'who-are-you',
+            value: 'someone-else'
+          }
+        }
+      ],
       fields: ['who-are-you'],
       locals: { showSaveAndExit: true },
       next: '/confirm', // TO BE UPDATED AS STEPS ARE ADDED
       backLink: 'current-progress'
     },
+    '/in-the-uk':{
+      behaviours: SaveFormSession,
+      forks: [
+        {
+          target: '/name',
+          condition: {
+            field: 'in-the-uk',
+            value: 'yes'
+          }
+        },
+        {
+          target: '/cannot-use-form',
+          condition: {
+            field: 'in-the-uk',
+            value: 'no'
+          }
+        }
+      ],
+      fields: ['in-the-uk'],
+      locals: { showSaveAndExit: true },
+      continueOnEdit: true,
+      next: '/name',
+      backLink: 'who-are-you'
```

Comment:
remove backlink property so that backs goes to the preceding page in the session (when user chooses immigration adviser or someone else, this will not be '/who-are-you'

Reply:
Changes done